static/app/views/automations/components/dataConditionNodes.tsx

```diff
@@ -92,6 +104,7 @@ export function useDataConditionNodeContext(): DataConditionNodeProps {
 
 type DataConditionNode = {
   label: string;
+  validate: ((condition: DataCondition) => string | undefined) | undefined;
```

Comment:
Can we make this an object argument (`({condition}) =>` in case we might want to pass more data in here at some point (like organization for feature flags, or something else)? Will make refactoring it easier in the future.

On that note, is `condition` sufficient? Might we want to compare the current condition to other conditions to see if they conflict? Or is something else responsible for that?

Reply:
conflicting conditions are handled separately, since we want to flag them immediately (rather than waiting until onSubmit), so `condition` is sufficient for this validation